network-api/networkapi/templates/fragments/responsive_screen_wide_image.html

```diff
@@ -55,6 +55,6 @@
       Assuming low probability for a high res screen (since it's an outdated browser).
     {% endcomment %}
         src="{{ img_1200.url }}"
-        alt=""
+        alt="{% trans "Hero Image" %}"
```

Comment:
Hi @robdivincenzo! 

This template is currently only being used as a hero image for the following templates:
- `custom_hero.html`
- `publication_hero.html`
- `app_install_hero.html`

Based on your feedback [here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12122#discussion_r1548161649), I have added the generic alt text of "hero image" so this `alt` field is not empty 👍